cupy_backends/cuda/api/driver.pyx

```diff
@@ -148,6 +82,10 @@ cpdef get_build_version():
         return 0
 
 
+cpdef bint _is_cuda_python():
+    return CUPY_USE_CUDA_PYTHON
```

Comment:
Just a thought, haven't figured it out completely: Maybe this should live in the "Constants (CuPy)" section in `runtime.pyx`, sitting side by side with `_is_hip_environment`?

Reply:
I'm thinking it's better to place this here considering `get_build_version()` returns the version of CUDA Python.